openid-4-verifiable-presentations-1_0.md

```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 The Verifier can use one of the following mechanisms to invoke a Wallet:
 
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
-- Domain-bound Universal Links/App link as an `authorization_endpoint`
-- no specific `authorization_endpoint`, End-User scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
+- URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
+
+For a cross device flow, either of the URL options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
+
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api).
```

Comment:
I'd put this one first and if possible word it so it sounds like the preferred option :)

Reply:
I think we might get some pushback against essentially recommending it so I'd rather consider that separately. Opened https://github.com/openid/OpenID4VP/issues/361 for that.

I think we can be clearer about the advantages though - @leecam @timcappalli could you check the new text please? Suggestions to improve it are welcome!

Reply:
we have a whole paragraph on the benefits that Tim helped re-write here: https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#appendix-A-4

Reply:
I've committed Kristina's suggestion to just refer to the existing text now.